### PR TITLE
pr: daily hello world

### PR DESCRIPTION
**Pull request by jandrecosta**
first commit for the daily screen

### CODE REVIEW

File: templates/views/daily.hbs [hunk 1]
```diff
@@ -0,0 +1 @@
+<div><p>hello world!</p></div>
```
Comment (ialexreis) [inline templates/views/daily.hbs]: Please start with body tag

**Review by ialexreis — CHANGES_REQUESTED** [review]
In order to develop this page with consistency, please review the following topics:

File: templates/views/daily.hbs (hunk 1 above)
Comment (jandrecosta) [reply templates/views/daily.hbs]: solved

**Comment by ialexreis** [comment]
Your pull request is awesome, I will acccept it !

**Pull request merged**